Sources/CodeEditTextView/Extensions/NSRange+/NSRange+TSRange.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  NSRange+TSRange.swift
+//  
+//
+//  Created by Khan Winter on 2/26/23.
+//
+
+import Foundation
+import SwiftTreeSitter
+
+extension NSRange {
+    var tsRange: TSRange {
+        return TSRange(points: .zero..<(.zero),
+                       bytes: (UInt32(self.location) * 2)..<(UInt32(self.location + self.length) * 2))
```

Comment:
### [CodeStyle]: 

Maybe try to not indent to much. In the `CodeEdit` repo this is not allowed by SwiftLint. We should keep it consistent.

```suggestion
        return TSRange(
            points: .zero..<(.zero),
            bytes: (UInt32(self.location) * 2)..<(UInt32(self.location + self.length) * 2)
        )
```